composer.json

```diff
@@ -29,7 +29,7 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.4",
```

Comment:
We don't actually want to merge a change to this value as part of this PR but including this change here temporarily should enable discovery of composer-detectable PHP version dependency problems.